aquifers/AQUFLUX-01.DATA

```diff
@@ -19,289 +18,273 @@ RUNSPEC
+--   REF.PRES.   COMPR.
+ROCK
+      277.0      6.11423e-05 /
 
-PVCDO
-    277.0        1.001      0.0          835.0    0.0
-/
+INCLUDE
+ 'include/AQUNUM-SGOF.inc' /
 
-ROCK
-      277.0     4.84E-5 /
+INCLUDE
+ 'include/AQUNUM-SWOF.inc' /
 
-DENSITY
-      860.04 1033.0    0.853  /  Justert 22/7
+-- =================================================================================================================================
+-- 
+-- REGIONS SECTION 
+-- 
+-- =================================================================================================================================
+REGIONS
 
-NOECHO
+EQUALS
+ 'EQLNUM' 1 /
+ 'FIPNUM' 1 /
+/
+ 
 -- =================================================================================================================================
 -- 
 -- SOLUTION SECTION 
 -- 
 -- =================================================================================================================================
 SOLUTION
 
+RPTRST
+  'BASIC = 2' 'PBPD' 'FIP' /
+
 EQUIL
--- Datum    P     woc     Pc   goc    Pc  Rsvd  Rvvd
-   2525.   270   2600    0.0  2200.  0.0   1*   0   0/
+-- Datum    P     woc       Pc   goc     Pc  Rsvd  Rvvd
+ 2585.00  285.00  2585.00   0.0 1800.00  0.0   1    1   0 /  
 
-AQUFLUX
- 1 1.0 /
- 2 1.0 /
-/
+RSVD
+ 2000.0  122.30
+ 2585.0  122.30 /
+
+
+RVVD
+ 2000.0  1E-4
+ 2585.0  1E-4 /
 
 
 AQUANCON
 -- Aq#     I1 I2  J1   J2  K1 K2 FACE
-  1     1   2    1   1   1   5  I-    0.8         1*         NO/
-  2     19  20   1   1   1   5  I+    1.         1*         NO/
+     1     1  8      3    3	3   3	'J-'	  1.00      1  /
 /
 
+AQUFLUX
+ 1 0.10 /
+/
 
 
-DEBUG
-12 /
-
-
-RPTRST
-BASIC=2 KRO KRW KRG PRES /
-
-RPTSOL
-FIP=3  SWAT /
 
 -- =================================================================================================================================
 -- 
 -- SUMMARY SECTION 
 -- 
 -- =================================================================================================================================
 SUMMARY
---
---       EXPORT STANDARD SUMMARY VARIABLE VECTORS TO FILE
---
-ALL
---
---       ACTIVATE EXCEL SUMMARY OUTPUT FILE OPTION     
---
-EXCEL
+
+TCPU
+
+FOPR
+
+WOPR
+/
+
+WBHP
+/
+
+WWCT
+/
+
+WGOR
+/
+
+FPR
 
 -- ---------------------------------------------------------------------------------------------------------------------------------
--- ANALYTICAL AQUIFER INFLUX VARIABLES                                                         
+-- NUMERICAL AQUIFER INFLUX VARIABLES                                                        
 -- ---------------------------------------------------------------------------------------------------------------------------------
 --
---       ANALYTICAL AQUIFERS(FIELD)
+--       NUMERICAL AQUIFER (FIELD)
 --
-FAQR
-FAQT
+FNQR
+FNQT
 --
---       ANALYTICAL AQUIFERS
+--       NUMERICAL AQUIFER
 --
 AAQP
- 1 2 / 
- 
+ 1 /
 AAQR
- 1 2 /
- 
+ 1 /
 AAQT
- 1 2 /
-
+ 1 /	   
```

Comment:
The header says "numerical aquifer", but the section requests `AAQ*` summary vectors.  At least one of the two is questionable to me.  Given that `AQUFLUX` uses analytic aquifer connections I would think that `AAQ*` would be the appropriate in which case the headers should probably refer to "analytic aquifers" instead.

Reply:
Thanks @bska , my bad. I started out by copying an numerical aquifer case and forgot to change this. Updated now